plugins/rect/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = () => ({ addUtilities, e, theme, variants }) => {
+  const pluginConfig = theme('rect', {})
+  const pluginVariants = variants('rect', [])
+
+  const utilities = Object.entries(pluginConfig).map(obj => {
+    // obj = ['target', ['44px', '44px']]
+    let valueIsArray = Array.isArray(obj[1])
```

Comment:
comment intentional?
do  you think we need to be checking the config structure? feels like we don't do that in any other plugin

Reply:
Comment was intentional. Re verifying the value is an array, yeah hard to know how much checking we should be doing 🤷‍♂️